### DISCUSSION TITLE
23.10.24 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#42] [Daily Scrum]
.

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - PR 머지
    - 소셜 로그인 테스트 작성법 학습
- 내일은 어떤 일을 하나요?
    - 코드 리뷰
    - 소셜 로그인 테스트 코드 작성

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - PR 작성 & 피드백
  - 테스트 코드 학습
- 내일은 어떤 일을 하나요?
  - PR 수정
  - 리팩토링 or 비동기 고민

**Comment by kdmstj** [comment]
- 오늘은 어떤 일을 했나요?
    - 면접 및 스터디 참여,,🥲
- 내일은 어떤 일을 하나요?
    - (서비스/컨트롤러 테스트 제외) 코드 PR
    - 우재님 코드 리뷰
    - 플레이리스트 음악 추가
